### PR TITLE
Add Note REs with healing links for Communal Healing and Sap Life

### PR DESCRIPTION
**Pull request by simonward**

### CODE REVIEW

File: static/lang/re-en.json [hunk 1]
```diff
@@ -1739,6 +1739,9 @@
                 "Prompt": "Select a clan weapon."
             },
             "Cleric": {
+                "CommunalHealing": {
+                    "Note": "When you cast the <em>heal</em> spell to heal a single creature, choose another willing living creature within range of your <em>heal</em> to regain [[/r (@item.rank)[vitality,healing]]] Hit Points."
```
Comment (stwlam) [inline static/lang/re-en.json:1743]: What does this look like in the chat log?

Comment (simonward) [reply static/lang/re-en.json:1743]: Screen Shot 2024-01-27 at 11 23 02 PM

Do you want me to remove the vitality type?

Comment (stwlam) [reply static/lang/re-en.json:1743]: You can specify a `shortLabel` option now in `@Damage` expression.

Comment (simonward) [reply static/lang/re-en.json:1743]: Since it's healing, it's `[[/r ]]` rather than `@Damage[]`.

Comment (simonward) [reply static/lang/re-en.json]: I went ahead and removed the vitality type.

File: static/lang/re-en.json [hunk 2]
```diff
@@ -1739,6 +1739,9 @@
                 "Prompt": "Select a clan weapon."
             },
             "Cleric": {
+                "CommunalHealing": {
+                    "Note": "When you cast the <em>heal</em> spell to heal a single creature, choose another willing living creature within range of your <em>heal</em> to regain [[/r (@item.rank)[healing]]] Hit Points."
```
Comment (stwlam) [inline static/lang/re-en.json:1743]:
```suggestion
                    "Note": "When you cast the <em>heal</em> spell to heal a single creature, choose another willing living creature within range of your <em>heal</em> to regain @Damage[@item.rank[vitality,healing]|shortLabel] Hit Points."
```

**Pull request merged**